app/views/flow/index.html.erb

```diff
@@ -47,7 +47,7 @@
     <td style="text-align: right;">
       <%= '%s - <b>%s</b>'.html_safe % [pluralize(order.line_items.sum(:quantity), 'item'), order.flow_total] %>
       <br/>
-      <small><%= order.flow_experience.key %></small>
+      <small><%= order.flow_order ? order.flow_order.experience[:key] : Flow.base_country %></small>
```

Comment:
seems odd that in one case we show an experience key, and in another the base country. If displaying a flow order, I think we want to show the order destination country here so that we always display country